gguf-py/gguf/constants.py

```diff
@@ -178,6 +178,28 @@ class Adapter:
         TYPE       = "adapter.type"
         LORA_ALPHA = "adapter.lora.alpha"
 
+    class Vision:
+        # only support vision.type = "clip" for now
```

Comment:
Probably better to be very specific here you are supporting ViT. HuggingFace has also moved away from calling every generic terms. Also I don't think the purpose is to support actual CLIP inference.

Reply:
Yeah, agree. I think for now it's safer to call this `clip-vit` to reflect the base implementation is `openai/clip-vit-*`

Atm it's quite complicated for me to drop the `clip_` prefix in all functions. But hey at least the file name is now `llama-vision.{cpp|h}`, which should reflect that we can support ViT and also other things to come in the future.